### ISSUE TITLE
Missing version 4.33 when use with FileStore.

### ISSUE THREAD

**Issue by vulehoangthanh** [kreait/firebase-php#339]
Hi , Kreait 
I follow your document , when i use FileStore feature, i install package follow your document. But v4.33 not found on your repo. 
```php
 {
   "require": {
     "google/cloud-firestore": "^1.8",
     "kreait/firebase-php": "^4.33"
   }
}
```

**Comment by jeromegamez** [comment]
v4.33 is not yet released, the current version is 4.32. If you want to use this feature, you should use `"kreait/firebase-php": "dev-master"` at the moment.

**Comment by vulehoangthanh** [comment]
Yes tks, but i think author should update document "kreait/firebase-php":"^4.33" to "kreait/firebase-php": "dev-master"
